swimster-ui/src/hooks/useListingDetail.js

```diff
@@ -27,7 +28,16 @@ export const useListingDetail = listingId => {
             setIsFetching(false)
         }
 
-        if (isAuthenticated) fetchListing()
+        const fetchImagesFromListing = async () => {
+            const { data, error } = await apiClient.fetchImagesFromListing(listingId)
+            if (error) setError(error)
+            if (data?.listingImages) setListingImages([...data.listingImages])
+        }
+
+        if (isAuthenticated) {
+            fetchListing()
+            fetchImagesFromListing()
```

Comment:
why do you create two fetches for both and not just one fetch method for the whole thing?

Reply:
true, will combine it now